src/components/HdIcon.vue

```diff
@@ -53,6 +67,27 @@ export default {
 
       return svg;
     },
+    addAccessibilityTags(svg) {
+      const randId = Math.floor(Math.random() * 1000);
+      const attrs = [];
+      if (this.title) {
+        const titleId = `${this.id || randId}-title`;
+        const titleTag = document.createElementNS('http://www.w3.org/2000/svg', 'title');
+        titleTag.innerHTML = this.title;
```

Comment:
```suggestion
        titleTag.textContent = this.title;
```

`innerHTML` can make the projects vulnerable to XSS attacks.